<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModrinthModBrowser</class>
 <widget class="QMainWindow" name="ModrinthModBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search mod on Modrinth...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>131</y>
      <width>93</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Mod Loader:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="sortSelect">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>80</y>
      <width>125</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Relevance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Downloads</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Follows</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Newest</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Updated</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="loaderSelect">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>131</y>
      <width>102</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Mod">
    <property name="title">
     <string>&amp;Mod</string>
    </property>
    <widget class="QMenu" name="menuDownload">
     <property name="title">
      <string>Download</string>
     </property>
     <property name="icon">
      <iconset theme="download">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <addaction name="actionOpen_Modrinth_Mod_Dialog"/>
    <addaction name="menuDownload"/>
    <addaction name="actionOpen_Website_Link"/>
    <addaction name="actionCopy_Website_Link"/>
   </widget>
   <widget class="QMenu" name="menu_Modrinth">
    <property name="title">
     <string>M&amp;odrinth</string>
    </property>
    <widget class="UnclosedMenu" name="menuSelect_Game_Version">
     <property name="styleSheet">
      <string notr="true">QMenu { menu-scrollable: 1; }</string>
     </property>
     <property name="title">
      <string>Select Game Version</string>
     </property>
    </widget>
    <widget class="UnclosedMenu" name="menuSelect_Category">
     <property name="styleSheet">
      <string notr="true">QMenu { menu-scrollable: 1; }</string>
     </property>
     <property name="title">
      <string>Select Category</string>
     </property>
    </widget>
    <addaction name="menuSelect_Game_Version"/>
    <addaction name="menuSelect_Category"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="menu_Mod"/>
   <addaction name="menu_Modrinth"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="MenuToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="searchBar">
   <property name="windowTitle">
    <string>Search Bar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="categoryToolBar">
   <property name="windowTitle">
    <string>Categories</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="versionToolBar">
   <property name="windowTitle">
    <string>Game Versions</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCopy_Website_Link">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy Website Link</string>
   </property>
  </action>
  <action name="actionOpen_Website_Link">
   <property name="text">
    <string>Open Website Link</string>
   </property>
  </action>
  <action name="actionOpen_Modrinth_Mod_Dialog">
   <property name="icon">
    <iconset resource="../../images/image.qrc">
     <normaloff>:/image/modrinth.svg</normaloff>:/image/modrinth.svg</iconset>
   </property>
   <property name="text">
    <string>Open Modrinth Mod Dialog</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuToolBar</class>
   <extends>QToolBar</extends>
   <header location="global">ui/menutoolbar.h</header>
  </customwidget>
  <customwidget>
   <class>UnclosedMenu</class>
   <extends>QMenu</extends>
   <header location="global">util/unclosedmenu.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/image.qrc"/>
 </resources>
 <connections/>
</ui>
